OculusLibrary.Tests/OculusLibrary.Tests.csproj

```diff
@@ -72,12 +72,21 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="OculusWebsiteParserTests.cs" />
+    <Compile Include="OculusApiScraperTests.cs" />
     <Compile Include="PathNormaliserTests.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Asgard%27s Wrath.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
     <None Include="packages.config" />
+    <Content Include="Sprint Vector.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Vader Immortal.json">
```

Comment:
Tests currently fail for me, as this isn't checked into `git`

Reply:
Oops. Had that lined up as a future third test case, but then thought 2 of the same test was enough.